src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -67,18 +66,17 @@ const getInstallationsWithAdmin = async (
 	gitHubUserClient: GitHubUserClient,
 	log: Logger,
 	login: string,
-	installations: Octokit.AppsListInstallationsForAuthenticatedUserResponseInstallationsItem[] = [],
-	jiraHost: string,
-	gitHubAppId?: number
+	installations: Octokit.AppsListInstallationsForAuthenticatedUserResponseInstallationsItem[] = []
 ): Promise<InstallationWithAdmin[]> => {
 	return await Promise.all(installations.map(async (installation) => {
 		const errors: Error[] = [];
-		const gitHubClient = await createInstallationClient(installation.id, jiraHost, log, gitHubAppId);
 
-		const numberOfReposPromise = await gitHubClient.getNumberOfReposForInstallation().catch((err) => {
-			errors.push(err);
-			return 0;
-		});
+		const numberOfReposPromise = await gitHubUserClient.getNumberOfReposForInstallation(installation.id)
```

Comment:
/nit
Is the return a `promise` or `number`?

Reply:
The return is a number. Cleaned up the `Promise.all` by using `await`.

Reply:
This has been reverted